aws-infra/cloudfront.py

```
@@ -0,0 +1,411 @@
+if [ -f ".nvmrc" ]; then
+  nvm use
+else
+  nvm use default || nvm use
+fi
+
+yarn install
+yarn build
+"""
+
+    pulumi.log.info("Building UI inside Pulumi: cd ui && nvm use && yarn install && yarn build")
+    subprocess.run(["bash", "-lc", cmd], check=True)
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)